mne_bids_pipeline/tests/test_documented.py

```diff
@@ -209,3 +211,15 @@ def ignore_unknown(self, node: yaml.Node) -> None:
     assert tests == examples, "CircleCI tests != docs/mkdocs.yml Examples"
     assert tests == dataset_names, "CircleCI tests != tests/datasets.py"
     assert tests == test_names, "CircleCI tests != tests/test_run.py"
+
+
+def test_config_template_valid(tmp_path: Path, monkeypatch: pytest.MonkeyPatch) -> None:
+    """Ensure our config template is syntactically valid (importable)."""
+    fpath = tmp_path / "foo.py"
+    create_template_config(fpath)
+    monkeypatch.setenv("BIDS_ROOT", str(tmp_path))
+    # we need check=False because (at least):
```

Comment:
So I think it's okay for these to fail. So how about:

1. Run with `check=True` inside a `pytest.raises`
2. Fix the errors
3. Run with `check=True` (smoke test complete)

If you like the idea but not motivated to do it now, let's just add this as a `TODO` comment and merge!

Reply:
> it's okay for these to fail for the reasons you comment below

Agreed. But I do like the idea of catching and handling those (to make sure there aren't any other problems). To me it seems a bit redundant to then also tweak the config and run without error catching, but it's not hard or expensive so I'll add that too (Monday probably).

Reply:
OK all green! And I take back what I said about redundancy; I was forgetting that with `pytest.raises`, execution will just stop once a matching error is raised (unlike, say, `warnings.filterwarnings`). So if we don't change the `ch_types` value in the file, the problem with no `conditions` defined won't actually surface in the test.